Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    clang \
+    gcc \
+    libc6-dev \
+    make \
+    curl \ 
+    ca-certificates \
+    git
+
+RUN mkdir -p /opt/go_dist &&\
+    curl https://dl.google.com/go/go1.11.linux-amd64.tar.gz | tar -xz -C /opt/go_dist
```

Comment:
(Optional) Would it make sense to compare against a known good sha256 checksum here?

Reply:
That totally belongs there. I also updated to the latest versions of go and golangci-lint.